raiden/tests/utils/blockchain.py

```diff
@@ -247,10 +262,15 @@ def geth_create_blockchain(
 
     cmds = []
     for i, config in enumerate(nodes_configuration):
-        nodedir = os.path.join(base_datadir, config['nodekeyhex'])
+        # HACK: Use only the first 8 characters to avoid golang's issue
+        # #6895 (IPC bind fails with path longer than 108 characters).
```

Comment:
Very interesting. I was not aware of this limitation in golang. Maybe add the link to the issue in the comment?
https://github.com/golang/go/issues/6895